tr3am/app/Components/Offices/OfficeDetails/index.jsx

```diff
@@ -49,6 +50,20 @@ class OfficeDetails extends React.Component {
 
   resetStateFromProps() {
     this.setState({ ...this.props.office });
+    this.setState({ errorMessage: '' });
+  }
+
+  officeExists() {
+    const { offices } = this.props;
+    const specificOffice = (offices.find(x => x.city === this.state.city));
+    if (specificOffice != null &&
```

Comment:
Instead of if else statement return `specificOffice != null &&specificOffice.country === this.state.country &&
            specificOffice.address === this.state.address`